dockers/docker-database/critical_processes.j2

```diff
@@ -0,0 +1,5 @@
+{% if INSTANCES %}
+{%     for redis_inst, redis_items in INSTANCES.items() %}
+program:{{ redis_inst }}
```

Comment:
Instances I see there are 2 of them does that mean will add both to critical process list ? 

    "INSTANCES": {
        "redis":{
            "hostname" : "127.0.0.1",
            "port" : 6379,
            "unix_socket_path" : "/var/run/redis/redis.sock",
            "persistence_for_warm_boot" : "yes"
        },
        "redis_chassis":{
            "hostname" : "redis_chassis.server",
            "port": 6380,
            "unix_socket_path": "/var/run/redis-chassis/redis_chassis.sock",
            "persistence_for_warm_boot" : "yes"
        }
    },

Reply:
no, this take care of having correct instance

`  update_chassisdb_config -j $db_cfg_file_tmp -k -p $chassis_db_port`